Library/Homebrew/dev-cmd/irb.rb

```diff
@@ -76,7 +76,6 @@ def run
         if args.pry?
           Pry.config.should_load_rc = false # skip loading .pryrc
           Pry.config.history_file = "#{Dir.home}/.brew_pry_history"
-          Pry.config.memory_size = 100 # max lines to save to history file
```

Comment:
This is already the [default](https://github.com/pry/pry/blob/2f8b2570f247b8cf074b6beea1aa62e6f8dd8f30/lib/pry/config.rb#L198).